integration/client-cli/admin-cli/src/main/java/org/keycloak/client/cli/common/BaseConfigCredentialsCmd.java

```diff
@@ -222,6 +224,7 @@ public String help() {
         out.println("    --keypass PASSWORD      Key password (prompted for if not specified, --keystore is used without --storepass, and KC_CLI_KEY_PASSWORD");
         out.println("                            otherwise defaults to keystore password)");
         out.println("    --alias ALIAS           Alias of the key inside a keystore (defaults to the value of ClientId)");
+        out.println("    --status                Check the validity of the connection");
```

Comment:
@shawkins My understanding is the command should not update the config but rather it should. check the status of the current tokens which are present.
Moreover, if we call the parent process method first it will always expect the server and other arguments to be populated. Otherwise it will throw an exception.

Reply:
@keshavprashantdeshpande it would require detecting if anything else were set and running the super method if anything is. That would look like:

```
if (!super.nothingToDo()) {
  super.process();
}
if (status) {
  status check ...
}
```

Alternatively you'll have to at least docuement that the other options are ignored when using the status flag - you could even make the parsing of the options exclusive, but that seems like too much work.

Reply:
> I think calling the super method would defeat the purpose of status command and I would like to go with alternative implementation.

I don't quite follow this concern. From what is shown in the previous comment, the super would only be called if non-status options are present.

> I have documented the changes. Please feel free to review it.

Thanks, that seems fine as well. I'll call the possible behaviors in a review comment. If anyone else wants the other behavior, we'll go that direction instead.